src/test/java/org/folio/rest/impl/AutomatedPatronBlocksAPITest.java

```diff
@@ -336,10 +348,11 @@ private void validateRecallOverdueByMaximumNumberOfDaysBlockResponse(int dueDate
     OpenLoan overdueLoan = new OpenLoan()
       .withLoanId(randomId())
       .withRecall(true)
+      .withItemLost(false)
```

Comment:
Does that make sense to move OpenLoan creation to separate method with due-date parameter?

Reply:
Done, thanks.